src/cryptoadvance/specter/controller.py

```diff
@@ -595,6 +613,12 @@ def new_device():
     app.specter.check()
     return render_template("device/new_device.jinja", specter=app.specter, rand=rand)
 
+@app.route('/hwi_new_device/')
+@login_required
+def hwi_new_device_xpubs():
```

Comment:
Or maybe we should rather splitup all hwi (including/excluding) rpc-stuff?

Reply:
I separated it now.